trace-sdk/src/main/java/io/bitrise/trace/data/management/formatter/cpu/ApplicationCpuDataFormatter.java

```diff
@@ -59,7 +55,7 @@ public static Metric createAppCpuMetric(final double cpuUsagePercent, @NonNull f
                                 .setUnit(percent)
                                 .setType(MetricDescriptor.Type.GAUGE_DOUBLE);
 
-        final TimeSeries appTimeSeries = createCpuTimeSeriesEntry(timestamp, (float) cpuUsagePercent);
```

Comment:
this caused an interesting side effect in the tests
![Screenshot 2021-04-28 at 12 24 14](https://user-images.githubusercontent.com/71286749/116405109-30273980-a827-11eb-8fb9-329c59b90313.png)

You can see i'm setting the value to be 12.3 - but because it's cast to a float it's then equal to 12.300000190734863. Was there a reason for this cast? L81 we're calling `setDoubleValue` so i figured it was safe to just remove.

Reply:
I do not see a reason for the cast, maybe the signature was different and the cast was needed, but after a change it is not required.
On the other hand I believe in L72 it should be a primitive double instead of Double, as null values are not acceptable in this case.

Reply:
Interesting - hopefully having a test with real data will catch this kind of issue in the future :-)

I'll update the method on L72 to ensure it's a double not a Double 👍